### PR TITLE
257 Enhancement on Pick a Winner comments retrieval.

### PR DESCRIPTION
**Pull request by cedricabuso**

### CODE REVIEW

File: winner/winner.js [hunk 1]
```diff
@@ -8,29 +8,74 @@ $(function () {
...
+                                title: e.snippet.topLevelComment.snippet.authorDisplayName,
+                                comment: e.snippet.topLevelComment.snippet.textDisplay,
+                                comment_id: e.id
+                            };
+
+                            if (e.replies) {
+                                replies = replies.concat(
+                                    _(e.replies.comments)
+                                        .map(function (j) {
+                                            var object = {
```
Comment (nhvu1988) [inline winner/winner.js]: return here

Comment (cedricabuso) [inline winner/winner.js]: @nhvu1988 Ahh, right. :)

**Comment by cedricabuso** [comment]
@nhvu1988 Updated. :)

**Comment by nhvu1988** [comment]
@cedricabuso we should ask Jesper in meeting today for using this implement or waiting for new one :)

**Comment by cedricabuso** [comment]
@nhvu1988 Ok. :)

@nhvu1988 and @ravenjohn 
Just want to let you know that I think Youtube API has changed again. When I was trying to implement Raven's solution I console.log the generated token and the one returned by Youtube API to check the nextPageToken, and this was what I saw:

```
CGQQAA //generated token
Cg0Q8PC6yffZxwIgACgBEhQIARCImcKdhdnHAhiw3L3C5cbGAhgCIGMowPu1y4b3kMW9AQ== //nextPageToken returned by youtube

**Comment by cedricabuso** [comment]
@nhvu1988 and @ravenjohn 
And it doesn't return any items. :)

Here's the payload:

```
{
            part: 'id,replies,snippet',
            videoId: videoId,
            key: htbt.config.google_api_key,
            maxResults: 100,
            pageToken: generate_token(100),
            order: 'time',
            fields: 'pageInfo,nextPageToken,items(id,'
                + 'snippet/topLevelComment/snippet/textDisplay,'
                + 'snippet/topLevelComment/snippet/authorDisplayName,'
                + 'replies/comments/id,'
                + 'replies/comments/snippet/textDisplay,'
                + 'replies/comments/snippet/authorDisplayName)'
        };

**Comment by ravenjohn** [comment]
@cedricabuso nice catch! It seems like they have different implementations of nextPageToken per entity. The token generator for getting channels is still working.

**Comment by ravenjohn** [comment]
@cedricabuso I suggest you keep the nexPageTokens and see if they change in any circumstance. If they are static like the channel's nextPageToken, we can just have a list of nextPageTokens. If given enough time, we can also study its pattern.

**Comment by cedricabuso** [comment]
@ravenjohn Alright! :)

@nhvu1988 So I guess we'll be keeping this implementation for the meantime. :)

**Comment by nhvu1988** [comment]
@cedricabuso ok :)

**Comment by cedricabuso** [comment]
@nhvu1988 Jesper said it's good to go. :)

**Pull request merged**